libcpuid/CMakeLists.txt

```diff
@@ -8,8 +8,11 @@ set(cpuid_sources
     asm-bits.c)
 
 if(WIN32)
-   list(APPEND cpuid_sources msrdriver.c)
+  list(APPEND cpuid_sources msrdriver.c)
 endif()
+if(UNIX)
+  add_compile_definitions(_GNU_SOURCE)
```

Comment:
Need to port this for autotools, e.g. add

```
if test "$build_windows" = "no"; then
	AM_CPPFLAGS="$AM_CPPFLAGS -D_GNU_SOURCE"
fi
```

before `AM_CONDITIONAL([WINDOWS], [test "$build_windows" = "yes"])` in `autoconf.ac`